airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationResolver.java

```diff
@@ -0,0 +1,39 @@
+package io.airbyte.integrations.destination.snowflake;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.destination.snowflake.SnowflakeDestination.DestinationType;
+import java.util.Map;
+
+public class SnowflakeDestinationResolver {
```

Comment:
can this class be reused for any other destination? if yes, can you possibly rename it to something more generic like 'DestinationIntermediateStoreResolver'

Reply:
Seems RedshiftDestination can use the same logic, but for now redshift does not support GCS, so for now we cannot reuse resolver in other place and i prefer to leave the name as is to avoid misunderstanding.